src/core/thread/mle_router.hpp

```diff
@@ -560,6 +560,15 @@ class MleRouter : public Mle
      *
      */
     void SetCcmEnabled(bool aEnabled) { mCcmEnabled = aEnabled; }
+
+    /**
+     * This method sets flag informing that thread protocol version check VR should be skipped
+     * when deciding if device should operate as router.
+     *
+     * @param[in]  aSkip  TRUE if the VR check should be skipped, FALSE otherwise.
+     *
+     */
+    void SetEnableProtoVersionCheckSkip(bool aSkip) { mThreadVersionCheckSkip = aSkip; }
```

Comment:
nit: reversing polarity would make this more consistent with other methods
```suggestion
    void SetThreadVersionCheckEnabled(bool aEnabled) { mThreadVersionCheckEnabled = aEnabled; }
```

Reply:
Done.